api.md

```diff
@@ -164,3 +164,15 @@ Methods:
 
 - <code>client.webhooks.<a href="./src/finch/resources/webhooks.py">unwrap</a>(\*args) -> object</code>
 - <code>client.webhooks.<a href="./src/finch/resources/webhooks.py">verify_signature</a>(\*args) -> None</code>
+
+# RequestForwarding
```

Comment:
The original intent here is for this name to be representative of the resource class, rather than a plain english version of the resource. If it's not a huge deal I think it would be helpful on our end to defer this, as changing this behavior conditionally will require some more time on our end.

Reply:
Okay that's cool